submissions/AnnaStoyano/memory-pair-game/index.js

```diff
@@ -0,0 +1,106 @@
+class Game {
+    constructor(cards) {
+        this.container = document.querySelector('.container');
+        this.cards = cards;
+    }
+
+    render() {
+        this.cards.forEach(card => {
+            this.container.insertAdjacentElement('beforeend', card.render());
+        });
+    }
+
+    checkOpenCards() {
+        let game = this;
+        let activeCardsNames;
+        this.container.addEventListener('click', function (event) {
+            event.stopPropagation();
+            let activeCards = game.cards.filter(card => card.cardForDisplay.classList.contains('active'));
+            if (activeCards.length == 2) {
+                activeCardsNames = activeCards.map(card => card.cardForDisplay.attributes['data-name'].value);
+                setTimeout(function () {
+                    if (activeCardsNames[0] == activeCardsNames[1]) {
+                        activeCards.forEach(card => card.cardForDisplay.classList.add('opened'));
+                    }
+                    activeCards.forEach(card => card.cardForDisplay.classList.remove('active'));
+                    setTimeout(game.checkWin.bind(game), 700); /* In order to card has time to change background color before confirm*/
+                }, 700);
+            }
+        });
+    }
+
+    shuffleCard() {
+        this.cards.sort(() => 0.5 - Math.random());
+    }
+
+    startGame() {
+        this.shuffleCard();
+        this.render();
+        this.checkOpenCards();
+        this.checkWin();
+    }
+
+    checkWin() {
+        let openCards = game.cards.filter(card => card.cardForDisplay.classList.contains('opened'));
+        if (openCards.length == this.cards.length) {
+            let restart = confirm('You WIN!\nRestart Game?');
```

Comment:
const here as well. Also, you can move this string to the global constant in the constructor